.github/workflows/aws_k8s_terratest.yml

```diff
@@ -47,9 +47,12 @@ jobs:
           go-version: 1.15.5
         id: go
 
-      - name: Clone scalar-terraform-examples
-        run: |
-          git clone -b main --depth 1 https://github.com/scalar-labs/scalar-terraform-examples modules
+      - name: Checkout scalar-terraform-examples
+        uses: actions/checkout@v2
+        with:
+          repository: scalar-labs/scalar-terraform-examples
+          ref: main
+          path: ./test/modules
```

Comment:
Just a question. Not sure, but `path` specifies `./test/modules` and is it OK to copy to `./modules/aws/${d}/;` in the next command?
Is it OK since working-directory is `./test`?

Reply:
Good point!

It's OK, Because input path is always relative to `$GITHUB_WORKSPACE`.

ref: 
https://github.com/actions/checkout#whats-new

```
- Improved layout
The input path is always relative to $GITHUB_WORKSPACE
```

Reply:
Thanks! 👍